manifests/install.pp

```diff
@@ -17,17 +17,18 @@
   $package_options        = undef,
 ) {
   include tomcat
+  $_catalina_home = pick($catalina_home, $::tomcat::catalina_home, $name)
```

Comment:
`$::tomcat::catalina_home` always has a value in the current master branch which will make `$name` never choosable.